src/app/library/components/add-playlist.component.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components/native';
+
+import I18n from '../../utils/i18n';
+
+import { PlaylistContainer, PlaylistImage, InfoWrapper, PlaylistName } from './playlist-item.component';
```

Comment:
If you using same styled components, create file and import it from playlist-item.component and add-playlist.componetn